diff -up pulseaudio-13.0/src/pulsecore/atomic.h.5~ pulseaudio-13.0/src/pulsecore/atomic.h
--- pulseaudio-13.0/src/pulsecore/atomic.h.5~	2019-09-16 23:05:46.031493339 +0200
+++ pulseaudio-13.0/src/pulsecore/atomic.h	2019-09-16 23:06:21.983865077 +0200
@@ -65,13 +65,13 @@ static inline void pa_atomic_store(pa_at
 #else
 
 static inline int pa_atomic_load(const pa_atomic_t *a) {
-    __sync_synchronize();
+    asm volatile("": : :"memory");
     return a->value;
 }
 
 static inline void pa_atomic_store(pa_atomic_t *a, int i) {
     a->value = i;
-    __sync_synchronize();
+    asm volatile("": : :"memory");
 }
 
 #endif
@@ -123,13 +123,13 @@ static inline void pa_atomic_ptr_store(p
 #else
 
 static inline void* pa_atomic_ptr_load(const pa_atomic_ptr_t *a) {
-    __sync_synchronize();
+    asm volatile("": : :"memory");
     return (void*) a->value;
 }
 
 static inline void pa_atomic_ptr_store(pa_atomic_ptr_t *a, void *p) {
     a->value = (unsigned long) p;
-    __sync_synchronize();
+    asm volatile("": : :"memory");
 }
 
 #endif
