diff -up pulseaudio-16.0/src/pulsecore/atomic.h.7~ pulseaudio-16.0/src/pulsecore/atomic.h
--- pulseaudio-16.0/src/pulsecore/atomic.h.7~	2022-05-28 17:18:37.000000000 +0200
+++ pulseaudio-16.0/src/pulsecore/atomic.h	2022-05-28 23:01:05.356747662 +0200
@@ -66,13 +66,13 @@ static inline void pa_atomic_store(pa_at
 #else
 
 static inline int pa_atomic_load(const pa_atomic_t *a) {
-    __sync_synchronize();
+    asm volatile("": : :"memory");
     return a->value;
 }
 
 static inline void pa_atomic_store(pa_atomic_t *a, int i) {
     a->value = i;
-    __sync_synchronize();
+    asm volatile("": : :"memory");
 }
 
 #endif
@@ -124,13 +124,13 @@ static inline void pa_atomic_ptr_store(p
 #else
 
 static inline void* pa_atomic_ptr_load(const pa_atomic_ptr_t *a) {
-    __sync_synchronize();
+    asm volatile("": : :"memory");
     return (void*) a->value;
 }
 
 static inline void pa_atomic_ptr_store(pa_atomic_ptr_t *a, void *p) {
     a->value = (uintptr_t) p;
-    __sync_synchronize();
+    asm volatile("": : :"memory");
 }
 
 #endif
