diff -up pulseaudio-14.99.1/src/daemon/pulseaudio.desktop.in.2~ pulseaudio-14.99.1/src/daemon/pulseaudio.desktop.in
--- pulseaudio-14.99.1/src/daemon/pulseaudio.desktop.in.2~	2021-05-18 20:35:03.338758242 +0200
+++ pulseaudio-14.99.1/src/daemon/pulseaudio.desktop.in	2021-05-18 20:36:58.242701140 +0200
@@ -7,4 +7,4 @@ Terminal=false
 Type=Application
 X-GNOME-Autostart-Phase=Initialization
 X-GNOME-HiddenUnderSystemd=true
-X-KDE-autostart-phase=1
+NotShowIn=KDE;
