diff -up pulseaudio-13.0/src/pulsecore/atomic.h.omv~ pulseaudio-13.0/src/pulsecore/atomic.h
--- pulseaudio-13.0/src/pulsecore/atomic.h.omv~	2019-09-16 23:04:05.857138113 +0200
+++ pulseaudio-13.0/src/pulsecore/atomic.h	2019-09-16 23:04:39.454353493 +0200
@@ -22,6 +22,7 @@
 ***/
 
 #include <pulsecore/macro.h>
+#include <stdint.h>
 
 /*
  * atomic_ops guarantees us that sizeof(AO_t) == sizeof(void*).  It is
@@ -117,7 +118,7 @@ static inline void* pa_atomic_ptr_load(c
 }
 
 static inline void pa_atomic_ptr_store(pa_atomic_ptr_t *a, void* p) {
-    __atomic_store_n(&a->value, p, __ATOMIC_SEQ_CST);
+    __atomic_store_n(&a->value, (uintptr_t)p, __ATOMIC_SEQ_CST);
 }
 
 #else
