diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index 4fd192e..4b58565 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -173,6 +173,10 @@ load-module module-equalizer-sink
 .endif
 ])dnl
 
+.ifexists module-device-manager@PA_SOEXT@
+load-module module-device-manager
+.endif
+
 ### Make some devices default
 #set-default-sink output
 #set-default-source input
