diff -up pulseaudio-14.99.1/src/daemon/default.pa.in.1~ pulseaudio-14.99.1/src/daemon/default.pa.in
--- pulseaudio-14.99.1/src/daemon/default.pa.in.1~	2021-05-18 20:31:23.278952737 +0200
+++ pulseaudio-14.99.1/src/daemon/default.pa.in	2021-05-18 20:32:34.227534783 +0200
@@ -165,6 +165,18 @@ load-module module-filter-heuristics
 load-module module-filter-apply
 ])dnl
 
+ifelse(@HAVE_DBUS@, 1, [dnl
+.ifexists module-dbus-protocol@PA_SOEXT@
+load-module module-dbus-protocol
+.endif
+])dnl
+
+ifelse(@HAVE_FFTW@, 1, [dnl
+.ifexists module-equalizer-sink@PA_SOEXT@
+load-module module-equalizer-sink
+.endif
+])dnl
+
 ### Make some devices default
 #set-default-sink output
 #set-default-source input
